pkgs/by-name/ht/httplib/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  cmake,
+  fetchFromGitHub,
+  openssl,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "httplib";
+  version = "0.18.0";
+
+  src = fetchFromGitHub {
+    owner = "yhirose";
+    repo = "cpp-httplib";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-cR1yRqZ6hZeGtMhiW003zcN0d/f/v1gMMNiL0hA1r6I=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ openssl ];
+
+  strictDeps = true;
+
+  meta = {
+    homepage = "https://github.com/yhirose/cpp-httplib";
+    description = "C++ header-only HTTP/HTTPS server and client library";
+    changelog = "https://github.com/yhirose/cpp-httplib/releases/tag/${finalAttrs.src.rev}";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [
+      AndersonTorres
+      aidalgol
+    ];
```

Comment:
Could you please remove me from the maintainers list now that you're picking this up?

Reply:
As you wish